src/main/java/mike/Ui.java

```diff
@@ -45,20 +67,36 @@ public void showList(TaskList taskList) {
         formatInChatBox(output.toString());
     }
 
+    /**
+     * Informs user that the addition of the task is successful
+     *
+     * @param taskList the list of tasks
+     * @param task the task to be added to the taskList
+     */
     public void showAddSuccess(TaskList taskList, Task task) {
         formatInChatBox("Got it. I've added this task:\n  " + task + "\n Now you have "
                 + taskList.size() + " tasks in the list.");
     }
 
+    /**
+     * Informs user that the marking of the task is successful
+     * @param task the task to be marked asa done
+     */
     public void showMarkSuccess(Task task) {
         formatInChatBox("Nice! I've marked this task as done: \n" + task);
```

Comment:
Slight typo for as, but its pretty good overall! Keeps to the specified coding style!